fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -410,6 +412,7 @@ ACTOR static Future<Void> decodeBackupLogValue(Arena* arena,
 								logValue.param1 = logValue.param1.withPrefix(addPrefix, tempArena);
 							}
 							logValue.param2 = addPrefix == StringRef() ? allKeys.end : strinc(addPrefix, tempArena);
+							*tenantMapChanging = *tenantMapChanging || TenantAPI::tenantMapChanging(logValue);
```

Comment:
This trio of lines (update tenantMapChanging, push back result, adjust mutation size) could benefit in clarity from being in a lambda helped called `addResult(logValue)` or something.